quinn-proto/src/connection/mod.rs

```diff
@@ -784,11 +786,12 @@ impl Connection {
         // Send MTU probe if necessary
         if buf.is_empty() && self.state.is_established() {
             let space_id = SpaceId::Data;
-            let probe_size = match self
-                .path
-                .mtud
-                .poll_transmit(now, self.spaces[space_id].next_packet_number)
-            {
+            // FIXME: Doesn't account for skipped PNs
```

Comment:
How big of an issue is this?

Reply:
Whoops, stale comment, that was fixed by the new `peek` call.